doc/usage.rst

```diff
@@ -480,3 +480,56 @@ GitHub
 - *MERGE_REQUEST_ID* is the pull request number.
 
 .. _`personal access token`: https://github.blog/2013-05-16-personal-api-tokens/
+
+.. _traceability_jira_automation:
+
+--------------------
+Jira ticket creation
+--------------------
+
+Traceable items can be configured to create Jira tickets for. Duplication of tickets is avoided by querying Jira
+first for existing tickets based on the Jira project and ticket summary. Below is an example configuration:
```

Comment:
Can we make field choice (`summary`) as configurable in below configuration?

Reply:
Updated docs in 9dc42e940c7c5caf58b77a382207a7ccb800e356 and implementation in f739f7646fb80e22c1128bda067ace787b5ee470